source/connect.txt

```diff
@@ -0,0 +1,32 @@
+.. _scala-connect:
```

Comment:
[q] Should this page be accessed when selecting `Connect to MongoDB` in the ToC? It didn't show up for me but I see it if I manually enter the URL. Not sure if that is part of this PR, but I think it just needs to be added to snooty.toml.

Reply:
No ticket to add a landing page for this Connect section in the epic. Will most likely be added in a future piece of work!